CODEOWNERS

```diff
@@ -28,16 +28,19 @@
 *.yml @charliegracie
 
 # CMake
-*.cmake @youngar @charliegracie @rwy0717
-/cmake/ @youngar @charliegracie @rwy0717
-**/CMakeLists.txt @youngar @charliegracie @rwy0717
+*.cmake @youngar @charliegracie @rwy7
```

Comment:
For some reason it still complains about this handle, not sure why

Reply:
Who is "it" ?

@rwy7 : Do you still want to be referenced in the CODEOWNERS file?  (hi, btw :-) )

Reply:
"it" being github; when you go to the "Files Changed" tab, there's an error message at the top.

Reply:
Is the problem that `rwy7` is not in the [list of committers](https://github.com/orgs/eclipse/teams/technology-omr-committers)?

```
Unknown owner on line 31: make sure @rwy7 exists and has write access to the repository 
```

Reply:
Made the requested changes.